.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [master]
```

Comment:
```suggestion
    branches: master
```
Sorry this is so nitty, but I think brackets are not needed

Reply:
are you sure? I took this from here https://github.com/pre-commit/action

Reply:
also, my linter says "Incorrect type. Expected "array"." if I remove the brackets

Reply:
You're probably right on both counts, I was comparing with the config used by dask. Ignore me 😄

Reply:
:laughing: 

have had a look at dask's repo, I gather it's fine to write to both ways